### PR TITLE
feat: add new build service commands

### PR DESCRIPTION
**Pull request by smile37773**
---

This checklist is used to make sure that common guidelines for a pull request are followed.

### General Guidelines

- [ ] Have you run `azdev style <YOUR_EXT>` locally? (`pip install azdev` required)
- [ ] Have you run `python scripts/ci/test_index.py -q` locally?

For new extensions:

- [ ] My extension description/summary conforms to the [Extension Summary Guidelines](https://github.com/Azure/azure-cli/blob/dev/doc/extensions/extension_summary_guidelines.md).

### About Extension Publish

There is a pipeline to automatically build, upload and publish extension wheels.  
Once your pull request is merged into main branch, a new pull request will be created to update `src/index.json` automatically.  
You only need to update the version information in file setup.py and historical information in file HISTORY.rst in your PR but do not modify `src/index.json`.

### CODE REVIEW

**Comment by yonzhan** [comment]
Thank you for your contribution! We will review the pull request and get back to you soon.

File: src/spring/HISTORY.md [hunk 1]
```diff
@@ -1,6 +1,14 @@
 Release History
 ===============
-1.12.0 (WIP)
+1.13.0
+---
+* Add new command -- `az spring container-registry create` to craete container registry resource.
+* Add new command -- `az spring container-registry delete` to delete container registry resource.
+* Add new command -- `az spring container-registry list` to list all the container registry resources.
+* Add new command -- `az spring build-service update` to update build service.
+* Add new command -- `az spring build-service show` to show build service resource.
```
Comment (zhoxing-ms) [inline src/spring/HISTORY.md:9]:
```suggestion
* Add new command `az spring container-registry create` to craete container registry resource.
* Add new command `az spring container-registry delete` to delete container registry resource.
* Add new command `az spring container-registry list` to list all the container registry resources.
* Add new command `az spring build-service update` to update build service.
* Add new command `az spring build-service show` to show build service resource.
```

File: src/spring/setup.py [hunk 2]
```diff
@@ -16,7 +16,7 @@
 
 # TODO: Confirm this is the right version number you want and it matches your
 # HISTORY.rst entry.
-VERSION = '1.11.3'
+VERSION = '1.13.0'
```
Comment (zhoxing-ms) [inline src/spring/setup.py:19]: Can you explain why you need to skip version `1.12.0`?

Comment (smile37773) [reply src/spring/setup.py]: OK, use version 1.12.0 here.

**Review by zhoxing-ms — APPROVED** [review]

**Pull request merged**